powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/provider/PowerAuthEncryptionProvider.java

```diff
@@ -54,8 +54,8 @@ public PowerAuthEciesDecryptorParameters getEciesDecryptorParameters(String acti
             eciesDecryptorRequest.setEphemeralPublicKey(ephemeralPublicKey);
             PowerAuthPortV3ServiceStub.GetEciesDecryptorResponse eciesDecryptorResponse = powerAuthClient.getEciesDecryptor(eciesDecryptorRequest);
             return new PowerAuthEciesDecryptorParameters(eciesDecryptorResponse.getSecretKey(), eciesDecryptorResponse.getSharedInfo2());
-        } catch (RemoteException e) {
-            logger.warn("Get Ecies decryptor parameters call failed", e);
+        } catch (RemoteException ex) {
+            logger.warn("Get ECIES decryptor call failed, error: {}", ex.getMessage());
             throw new PowerAuthEncryptionException();
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.